lib/AST/Builtins.cpp

```diff
@@ -2249,9 +2281,16 @@ ValueDecl *swift::getBuiltinValueDecl(ASTContext &Context, Identifier Id) {
 
   case BuiltinValueKind::IsUnique:
   case BuiltinValueKind::IsUnique_native:
+  case BuiltinValueKind::BeginCOWmutation:
+  case BuiltinValueKind::BeginCOWmutation_native:
```

Comment:
I don't understand this since `getIsUniqueOperation` returns a single bool result

[EDIT] Oh, now I get it. Builtin.isUnique could be treated exactly like the new builtin, except that we can't assume there's an eventual end mutation marker. Could we use begin_cow_mutation for Builtin.isUnique anyway and just not worry that there will not be an end_cow_mutation?

This is a little tricky so it would be helpful to have a short comment

Reply:
Eventually we should convert all COW data structures in the stdlib to Begin/EndCOWMutation. So isUnique should go away anyway (except some uses in assert conditions).

Reply:
I think there will still be a public `isUniquelyReferenced` API though. The question is whether it should have a different implementation from CoW mutation